velox/docs/spark_functions.rst

```diff
@@ -60,32 +60,33 @@ for :doc:`all <functions/spark/coverage>` functions.
-    :spark:func:`ceil`                :spark:func:`log1p`               :spark:func:`split`                                                                                         
-    :spark:func:`chr`                 :spark:func:`lower`               :spark:func:`startswith`                                                                                    
-    :spark:func:`concat`              :spark:func:`ltrim`               :spark:func:`substring`                                                                                     
-    :spark:func:`contains`            :spark:func:`map`                 :spark:func:`subtract`                                                                                      
-    :spark:func:`csc`                 :spark:func:`map_filter`          :spark:func:`to_unix_timestamp`                                                                             
-    :spark:func:`divide`              :spark:func:`map_from_arrays`     :spark:func:`transform`                                                                                     
-    :spark:func:`element_at`          :spark:func:`md5`                 :spark:func:`trim`                                                                                          
-    :spark:func:`endswith`            :spark:func:`might_contain`       :spark:func:`unaryminus`                                                                                    
-    :spark:func:`equalnullsafe`       :spark:func:`multiply`            :spark:func:`unix_timestamp`                                                                                
-    :spark:func:`equalto`             :spark:func:`not`                 :spark:func:`upper`                                                                                         
-    :spark:func:`exp`                 :spark:func:`notequalto`          :spark:func:`xxhash64`                                                                                      
-    :spark:func:`filter`              :spark:func:`pmod`                :spark:func:`year`                                                                                          
+    :spark:func:`abs`                 :spark:func:`get_json_object`     :spark:func:`regexp_extract`          :spark:func:`bit_xor`                 :spark:func:`nth_value`
+    :spark:func:`acos`                :spark:func:`greaterthan`         :spark:func:`remainder`               :spark:func:`first` 
+    :spark:func:`acosh`               :spark:func:`greaterthanorequal`  :spark:func:`replace`                 :spark:func:`first_ignore_null` 
+    :spark:func:`add`                 :spark:func:`greatest`            :spark:func:`right`                   :spark:func:`last` 
+    :spark:func:`aggregate`           :spark:func:`hash`                :spark:func:`rlike`                   :spark:func:`last_ignore_null` 
```

Comment:
The community prefers a bulk update for this file (I guess, if not, it can easily cause code conflict). Let's remove this. Thanks!

Reply:
@PHILO-HE Thank you for the suggestion! The file has been reverted.